tests/rust/test_rust_backend.py

```
@@ -358,49 +395,68 @@
 
     def test_rust_backend_after_disposal(self, rust_backend):
         """Test rust backend behavior after disposal/cleanup."""
-        if hasattr(rust_backend, 'dispose'):
+        if hasattr(rust_backend, "dispose"):
             rust_backend.dispose()
-            if hasattr(rust_backend, 'compile'):
+            if hasattr(rust_backend, "compile"):
                 with pytest.raises(Exception):
```

Ruff (0.11.9)

Do not assert blind exception: `Exception`

(B017)